src/vtp/tally_contests.py

```diff
@@ -26,20 +26,21 @@
 See ../docs/tech/*.md for the context in which this file was created.
 """
 
-# Standard imports
-import sys
-import re
 # pylint: disable=wrong-import-position   # import statements not top of file
+# Standard imports
 import argparse
 import logging
-from logging import info, error
+import re
+import sys
+from logging import error, info
 
 # Local import
-from ballot import Ballot
-from election_config import ElectionConfig
-from common import Shellout
-from contest import Tally
-from exceptions import TallyException
+from utils.ballot import Ballot
+from utils.common import Shellout
+from utils.contest import Tally
+from utils.election_config import ElectionConfig
+from utils.exceptions import TallyException
```

Comment:
Ditto: use relative imports.

```suggestion
from .utils.ballot import Ballot
from .utils.common import Shellout
from .utils.contest import Tally
from .utils.election_config import ElectionConfig
from .utils.exceptions import TallyException
```